app/src/main/java/com/example/voyagerx/ui/fragments/landing/LandingPageFragment.kt

```diff
@@ -33,19 +41,36 @@ class LandingPageFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        if (!this::launchData.isInitialized) {
```

Comment:
Just confirming, I'm currently understanding this as:

"If launch data is NOT initialized, setupList()". 

Is that what you're aiming for? Its also the same boolean logic as line 33 where we show the spinner

Reply:
Yeah that was my goal, I'll consolidate this inside onViewCreated.